contracts/CPKFactory.sol

```diff
@@ -31,30 +24,21 @@ contract CPKFactory {
     function createProxyAndExecTransaction(
         address owner,
         address safeVersion,
-        uint256 saltNonce,
+        uint256 saltNonceSalt,
```

Comment:
I would just call it nonce ... not sure if it is really helpful to be that explicit here

Reply:
Calling it salt :P